modules/http4s/src/smithy4s/http4s/internals/SmithyHttp4sServerEndpoint.scala

```diff
@@ -125,7 +125,10 @@ private[smithy4s] class SmithyHttp4sServerEndpointImpl[F[_], Op[_, _, _, _, _],
   private val extractInput: (Metadata, Request[F]) ==> I = {
```

Comment:
Are we getting anything out of that kleisli? feels like it could be just a function.

Reply:
just removed it, good catch